docs/quick-start.md

```diff
@@ -3,22 +3,374 @@ title: Getting Started
 toc: true
 weight: 210
 ---
-# Quick Start Guide
+# Getting Started
 
 This quick start will demonstrate using Crossplane to deploy a portable MySQL
-database on a single cloud provider, Google Cloud Platform. It serves as an initial introduction to
-Crossplane, but only displays a small portion of its functionality.
+database on a single cloud provider, Google Cloud Platform (GCP). It serves as
+an initial introduction to Crossplane, but only displays a small portion of its
+functionality.
 
-The general steps for this example are as follows:
+In this guide we will:
 
-1. [Install Crossplane and the GCP stack](install-crossplane.md) into your Kubernetes cluster. 
-1. [Configure GCP credentials](cloud-providers/gcp/gcp-provider.md) in Crossplane.
-1. [Provision a MySQL database](quick-start-gcp.md) on GCP. 
+1. [Install Crossplane and the GCP stack](#install-crossplane).
+1. [Configure Crossplane to connect to GCP](#configure-crossplane-to-connect-to-gcp).
+1. [Provision a MySQL instance](#provision-a-mysql-instance).
+1. [Enable Dynamic Provisioning of MySQL databases](#enable-dynamic-provisioning).
+
+## Install Crossplane
+
+We'll start by installing Crossplane using [Helm]. You'll need a working
+Kubernetes cluster ([minikube] will do just fine). Crossplane is currently in
+alpha, so we'll use the `alpha` channel:
+
+```bash
+# Crossplane lives in the crossplane-system namespace by convention.
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+Once Crossplane is installed we'll need to install the GCP [stack]. Installing
+the GCP stack teaches Crossplane how to provision and manage things in GCP. You
+install it by creating a `ClusterStackInstall`:
+
+```yaml
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: ClusterStackInstall
+metadata:
+  name: stack-gcp
+  namespace: crossplane-system
+spec:
+  package: "crossplane/stack-gcp:master"
+```
+
+Save the above as `stack.yaml`, and apply it by running:
+
+```bash
+kubectl apply -f stack.yaml
+```
+
+We've now installed Crossplane with GCP support! Take a look at the guide to
+[installing Crossplane] for more installation options, and to learn how to
```

Comment:
nit: Take a look at the [Crossplane installation][installing Crossplane] for ...

Reply:
I think I had this worded slightly awkwardly because at some point it made the line endings align more pleasingly. :|